pkg/cvo/upgradeable.go

```diff
@@ -234,6 +247,126 @@ func (check *clusterManifestDeleteInProgressUpgradeable) Check() *configv1.Clust
 	return nil
 }
 
+func gateApplicableToCurrentVersion(gateName string, currentVersion string) (bool, error) {
+	var applicable bool
+	if ackVersion := adminAckGateRegexp.FindString(gateName); ackVersion == "" {
+		return false, fmt.Errorf("%s configmap gate name %s has invalid format; must begin with 'ack-X.Y'",
```

Comment:
If the regexp requires a `-` after the 4.y, we should mention that here^ too

Reply:
It's only checking for `ack-x.y` and ensuring we have a major/minor version number.